cli/cmd/event.go

```diff
@@ -132,6 +134,41 @@ time range.`,
 				cli.OutputHuman("\n")
 				cli.OutputHuman(entityTable)
 			}
+
+			cli.OutputHuman(
+				"\nFor further investigation of this event navigate to %s\n",
```

Comment:
@scottford-lw does this message sound correct?

Reply:
@afiune Wording seems 👍  to me.